quasar-core/src/main/java/co/paralleluniverse/fibers/instrument/QuasarURLClassLoaderHelper.java

```diff
@@ -23,16 +25,15 @@
 import java.net.URL;
 import java.net.URLClassLoader;
 import java.nio.ByteBuffer;
-import java.security.AccessControlContext;
-import java.security.AccessController;
-import java.security.CodeSigner;
-import java.security.PrivilegedExceptionAction;
+import java.security.*;
```

Comment:
Not good style to collapse import to a wildcard. IntelliJ setting for Java not up to date?

Reply:
It's not like anyone has checked any IntelliJ configuration files into this repository :man_facepalming:!